### ISSUE TITLE
pykalman/datasets/ not copied when pykalman is installed with pip

### ISSUE THREAD

**Issue by Naereen** [pykalman/pykalman#66]
Hi,

There must be a problem in the `setup.py` script, because when (fresh) installing pykalman (on a Ubuntu machine), all the python code is copied but not the `data/` and `descr/` folders of the `pykalman/datasets/` folder.

**Comment by yuzhao88** [comment]
Just had the same problem, and the .mat file from github cannot be loaded. Error message is ValueError: Unknown mat file type, version 100, 110

**Comment by Naereen** [comment]
2 months and no one replied yet?

Is the @pykalman project still maintained?!

**Comment by yuzhao88** [comment]
looks like not. I ran into multiple issues with the example code and couldn't get it running at all.

**Comment by Naereen** [comment]
Sadly.

**Comment by Naereen** [comment]
No one is maintaining pykalman?

**Comment by sergiobd** [comment]
Indeed, the dataset folder is not copied when installed with pip. However, if you replace the dataset folder from your pip install, by the repo's dataset folder, the example works.

**Comment by yurikleb** [comment]
Hello, are there any plans to make the installation work with PIP?
